src/features/import/components/ImportDialog/Configure/Configuration/GenderConfigRow.tsx

```diff
@@ -0,0 +1,132 @@
+import { ArrowForward, Delete } from '@mui/icons-material';
+import {
+  Box,
+  Button,
+  FormControl,
+  IconButton,
+  InputLabel,
+  MenuItem,
+  Select,
+  Typography,
+} from '@mui/material';
+import { FC, useState } from 'react';
+
+import messageIds from 'features/import/l10n/messageIds';
+import { Msg, useMessages } from 'core/i18n';
+import { Gender, genders } from '../../../../hooks/useGenderMapping';
+
+interface GenderConfigRowProps {
+  italic?: boolean;
+  numRows: number;
+  onSelectGender: (gender: Gender) => void;
+  onDeselectGender: () => void;
+  selectedGender: Gender | null;
+  title: string;
+}
+
+const GenderConfigRow: FC<GenderConfigRowProps> = ({
+  italic,
+  numRows,
+  onSelectGender,
+  onDeselectGender,
+  selectedGender,
+  title,
+}) => {
+  const messages = useMessages(messageIds);
+  const [mapping, setMapping] = useState(false);
+
+  const showSelect = mapping || selectedGender;
+
+  return (
+    <Box display="flex" flexDirection="column">
+      <Box display="flex">
+        <Box
+          alignItems="flex-start"
+          display="flex"
+          justifyContent="space-between"
+          paddingTop={1}
+          width="50%"
+        >
+          <Box display="flex" sx={{ wordBreak: 'break-all' }} width="100%">
+            <Typography fontStyle={italic ? 'italic' : ''}>{title}</Typography>
+          </Box>
+          <ArrowForward color="secondary" sx={{ marginRight: 1 }} />
+        </Box>
+        <Box
+          alignItems="flex-start"
+          display="flex"
+          paddingRight={1}
+          width="50%"
+        >
+          {!showSelect && (
+            <Button onClick={() => setMapping(true)}>
+              <Msg
+                id={
+                  messageIds.configuration.configure.genders
+                    .showGenderSelectButton
+                }
+              />
+            </Button>
+          )}
+          {showSelect && (
```

Comment:
This logic, to have a button in place of the Select, was created in the organization mapping because loading and rendering each Select with organisations took a really really long time. So this button was placed there to only load/render each select "on demand". With genders, since no loading of data is happening and it's a small set of values, i think it would be nice to remove the button and just show the select!

Reply:
Nice, I thought this was for a nicer UI. Removed 🧹